app/gateway/2.7.x/admin-api/consumer-groups/examples.md

```diff
@@ -0,0 +1,532 @@
+    [specific Service or Route](/hub/kong-inc/rate-limiting-advanced)
+    for more granular control.
+
+2. The plugin you just set up applies to all consumers in the cluster. Change
+the rate limiting configuration for the `JL` consumer group only, setting
+the limit to ten requests for every ten seconds:
+
+{% capture override %}
+{% navtabs codeblock %}
+{% navtab cURL %}
+```bash
+curl -i -X PUT http://{HOSTNAME}:8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced \
+  --data config.limit=10 \
+  --data config.window_size=10 \
+  --data config.retry_after_jitter_max=1
+```
+{% endnavtab %}
+{% navtab HTTPie %}
+```bash
+http PUT :8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced \
+  config.limit=10 \
+  config.window_size=10 \
+  config.retry_after_jitter_max=1
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+
+{{ override | indent | replace: " </code>", "</code>" }}
+
+    Response:
+
+    ```json
+    {
+      "config": {
+          "limit": [
+              10
+          ],
+          "retry_after_jitter_max": 1,
+          "window_size": [
+              10
+          ]
+      },
+      "consumer_group": "JL",
+      "plugin": "rate-limiting-advanced"
+    }
+    ```
+
+3. Check that it worked by looking at the `JL` consumer group object:
+
+{% capture check_group1 %}
+{% navtabs codeblock %}
+{% navtab cURL %}
+```bash
+curl -i -X GET http://{HOSTNAME}:8001/consumer_groups/JL
+```
+{% endnavtab %}
+{% navtab HTTPie %}
+```bash
+http :8001/consumer_groups/JL
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+
+{{ check_group1 | indent | replace: " </code>", "</code>" }}
+
+    Notice the `plugins` object in the response, along with the parameters that you
+    just set for the Rate Limiting Advanced plugin:
+
+    ```json
+    {
+        "consumer_group": {
+            "created_at": 1638915521,
+            "id": "8a4bba3c-7f82-45f0-8121-ed4d2847c4a4",
+            "name": "JL"
+        },
+        "consumers": [
+            {
+                "created_at": 1638915577,
+                "id": "8089a0e6-1d31-4e00-bf51-5b902899b4cb",
+                "type": 0,
+                "username": "DianaPrince",
+                "username_lower": "dianaprince"
+            }
+        ],
+        "plugins": [
+            {
+                "config": {
+                    "limit": [
+                        10
+                    ],
+                    "retry_after_jitter_max": 1,
+                    "window_size": [
+                        10
+                    ],
+                    "window_type": "sliding"
+                },
+                "consumer_group": {
+                    "id": "8a4bba3c-7f82-45f0-8121-ed4d2847c4a4"
+                },
+                "created_at": 1638916518,
+                "id": "b7c426a2-6fcc-4bfd-9b7a-b66e8f1ad260",
+                "name": "rate-limiting-advanced"
+            }
+        ]
+    }
+    ```
+
+## Remove consumer from group - group view
```

Comment:
Shall we show examples of deleting all consumers from a group and delete all groups for a consumer?
I think the API reference page has it but want to double check.

Reply:
Added.